Helper/Address.php

```diff
@@ -46,7 +46,7 @@ public function __construct(AdyenLogger $logger)
 
     // Regex to extract the house number from the street line if needed (e.g. 'Street address 1 A' => '1 A')
     const STREET_FIRST_REGEX = "/(?<streetName>[[:alnum:].'\- ]+)\s+(?<houseNumber>\d{1,10}((\s)?\w{1,3})?)$/u";
-    const NUMBER_FIRST_REGEX = "/^(?<houseNumber>\d{1,10}((\s)?\w{1,3})?)\s+(?<streetName>[a-zA-Z0-9.'\- ]+)/";
+    const NUMBER_FIRST_REGEX = "/^(?<houseNumber>\d{1,10}((\s)?\w{1,3})?)\s+(?<streetName>[A-Za-zÀ-ÖØ-öø-Ÿ0-9.'\- ]+)/";
```

Comment:
I think alphanumeric match is possible here too, try it together with the unicode modifier (`u`):
```suggestion
    const NUMBER_FIRST_REGEX = "/^(?<houseNumber>\d{1,10}((\s)?\w{1,3})?)\s+(?<streetName>[[:alnum:].'\- ]+)/u";
```

Reply:
Indeed we did the same for `STREET_FIRST_REGEX`

Reply:
@tnaber What do you think about my suggestion? I think it's better than using specific character ranges which might fail for other languages.